src/store/login/index.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
+/* eslint-disable arrow-body-style */
+/* eslint-disable prefer-template */
+export default {
+  state: {
+    fbToken: null,
+    jwtToken: null
+  },
+  getters: {
+    isLoggedIn(state) {
+      return (state.fbToken !== null) && (state.jwtToken !== null);
+    }
+  },
+  mutations: {
+    login(state, payload) {
+      state.fbToken = payload.fbToken;
+      state.jwtToken = payload.jwtToken;
+    },
+    logout(state) {
+      state.fbToken = null;
+    }
+  },
+  actions: {
+    login({ dispatch, commit }) {
+      return new Promise((resolve, reject) => {
+        window.FB.login((response) => {
+          if (response.authResponse) {
+            resolve(response.authResponse);
+          } else {
+            reject('User cancelled login authentication');
+          }
+        });
+      }).then((authResponse) => {
+        return dispatch('serverAuthentication', {
+          authResponse
+        });
+      }).then((serverResponse) => {
+        commit('login', serverResponse);
```

Comment:
It has been a convention to have a separate string mapping for other devs to see at a glance what mutations are available for use. Do you think we won't need that?

Reply:
KIV